core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -121,10 +125,77 @@ protected void doStop() throws Exception {
+        sslContextFactory.setKeyManagerPassword(config.getString(RestConfig.SSL_KEY_PASSWORD_CONFIG));
+        sslContextFactory.setKeyStoreType(config.getString(RestConfig.SSL_KEYSTORE_TYPE_CONFIG));
+
+        if (!config.getString(RestConfig.SSL_KEYMANAGER_ALGORITHM_CONFIG).isEmpty()) {
+          sslContextFactory.setSslKeyManagerFactoryAlgorithm(
+                  config.getString(RestConfig.SSL_KEYMANAGER_ALGORITHM_CONFIG));
+        }
+      }
+
+      sslContextFactory.setNeedClientAuth(config.getBoolean(RestConfig.SSL_CLIENT_AUTH_CONFIG));
+
+      if (!config.getString(RestConfig.SSL_ENABLED_PROTOCOLS_CONFIG).isEmpty()) {
+        sslContextFactory.setIncludeProtocols(config.getString(RestConfig.SSL_ENABLED_PROTOCOLS_CONFIG).split(","));
+      }
+
+      if (!config.getString(RestConfig.SSL_CIPHER_SUITES_CONFIG).isEmpty()) {
+        sslContextFactory.setIncludeCipherSuites(config.getString(RestConfig.SSL_CIPHER_SUITES_CONFIG).split(","));
+      }
+
+      if (!config.getString(RestConfig.SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG).isEmpty()) {
+        sslContextFactory.setEndpointIdentificationAlgorithm(
+                config.getString(RestConfig.SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG));
+      }
+
+      if (!config.getString(RestConfig.SSL_TRUSTSTORE_LOCATION_CONFIG).isEmpty()) {
+        sslContextFactory.setTrustStorePath(config.getString(RestConfig.SSL_TRUSTSTORE_LOCATION_CONFIG));
+        sslContextFactory.setTrustStorePassword(config.getString(RestConfig.SSL_TRUSTSTORE_PASSWORD_CONFIG));
+        sslContextFactory.setTrustStoreType(config.getString(RestConfig.SSL_TRUSTSTORE_TYPE_CONFIG));
+
+        if (!config.getString(RestConfig.SSL_TRUSTMANAGER_ALGORITHM_CONFIG).isEmpty()) {
+          sslContextFactory.setTrustManagerFactoryAlgorithm(
+                  config.getString(RestConfig.SSL_TRUSTMANAGER_ALGORITHM_CONFIG));
+        }
+      }
+
+      sslContextFactory.setProtocol(config.getString(RestConfig.SSL_PROTOCOL_CONFIG));
+      if (!config.getString(RestConfig.SSL_PROVIDER_CONFIG).isEmpty()) {
+        sslContextFactory.setProtocol(config.getString(RestConfig.SSL_PROVIDER_CONFIG));
+      }
+
+      NetworkTrafficServerConnector sslConnector = new NetworkTrafficServerConnector(server, sslContextFactory);
+      sslConnector.addNetworkTrafficListener(new MetricsListener(metrics, "jetty-https", metricTags));
+      sslConnector.setPort(getConfiguration().getInt(RestConfig.PORT_HTTPS_CONFIG));
+      server.addConnector(sslConnector);
+      log.info("Added https connector.");
+    }
+
+    if (server.getConnectors().length < 1) {
+      log.error("No connectors were added to Jetty, meaning http and https are disabled. " +
+               "This is likely a configuration error. " + RestConfig.REST_PROTOCOL_CONFIG +
+               " is likely not configured correctly. Current value: " +
+               config.getString(RestConfig.REST_PROTOCOL_CONFIG));
```

Comment:
Normally, errors result in an exception, otherwise warn can be used. But maybe there's a good reason to do it differently here.